.travis.yml

```diff
@@ -67,17 +67,11 @@ script:
   - bundle exec rails assets:precompile
   - './cc-test-reporter before-build'
   - if [ "$KNAPSACK_PRO_CI_NODE_INDEX" == "0" ]; then yarn test --colors; fi
+  - if [ "$KNAPSACK_PRO_CI_NODE_INDEX" == "0" ]; then bin/e2e-ci; fi
```

Comment:
@maestromac has another PR coming that I'll get into here to rework this. See #12442.